samples/stand-alone-samples/shadow/shadow_sync/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.1)
+project(shadow-sync CXX)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_INSTALL_PREFIX}/lib/cmake")
+
+file(GLOB SHADOW_SYNC_SRC
+       "*.cpp"
+)
+
+add_executable(${PROJECT_NAME} ${SHADOW_SYNC_SRC})
+
+set_target_properties(${PROJECT_NAME} PROPERTIES
+    LINKER_LANGUAGE CXX
+    CXX_STANDARD 11)
+
+if(COMMAND cmake_policy)
+    cmake_policy(SET CMP0003 NEW)
+endif(COMMAND cmake_policy)
```

Comment:
Is this necessary? I don't really understand cmake policies, reading about them now...


It says [here](https://cmake.org/cmake/help/latest/manual/cmake-policies.7.html#introduction):
> The cmake_minimum_required() command does more than report an error if a too-old version of CMake is used to build a project. It also sets all policies introduced in that CMake version or earlier to NEW behavior

It looks like CMP0003 was [introduced in cmake 2.6](https://cmake.org/cmake/help/latest/manual/cmake-policies.7.html#policies-introduced-by-cmake-2-6). Since we're requiring 3.1, should we already get the NEW behavior?

Reply:
yeah, it's still a copy pasted, and I haven't go through the detail. Will remove